plugins/kubernetes/app/models/kubernetes/role.rb

```diff
@@ -121,8 +121,8 @@ def defaults
       }
     end
 
-    def role_config_file
-      return unless raw_template = project.repository.file_content(config_file, 'HEAD', pull: false)
+    def role_config_file(reference)
```

Comment:
nope, only called in 2 locations ... makes it more obvious that it should not be randomly picked